### ISSUE TITLE
hyprland/language does not obey format on bar start

### ISSUE THREAD

**Issue by judemille** [Alexays/Waybar#1699]
The `hyprland/language` module does not obey the `format` option on the initialization, only after it receives the first event. This can be confirmed by looking at the code.

**Comment by hekatonkheirex** [comment]
I can confirm, it crashes when having "format": "{short}" or "{shortDescription}"

**Comment by judemille** [comment]
Crashes are not the issue at hand here, I suggest making a new issue if you're getting them.
